src/event_handler.py

```diff
@@ -67,6 +68,8 @@ def store_queued_events(_, __):
             if event.event_type == 'session_event' and event.details.get('session_event_type') == 'fraud_detected':
                 write_fraud_event_to_database(event, db_connection)
                 logger.info('Stored fraud event: {0}'.format(event.event_id))
+                send_to_fraud_logger(event)
```

Comment:
I think we need to be filtering `event` and in particular `event.details` for PII, but I'll try and confirm the details before we think about actioning this.

- [ ] check for personally identifiable data, and whitelist non-PII data